nnpackage/schema/circle_schema.fbs

```diff
@@ -224,8 +233,11 @@ table Tensor {
 // ones, but not by much. Moreover, while custom operators accept an opaque
 // object containing configuration parameters, builtins have a predetermined
 // set of acceptable options.
-
-enum BuiltinOperator : ubyte {
+// LINT.IfChange
+enum BuiltinOperator : int32 {
+  BCQ_GATHER = -4,
+  BCQ_FULLY_CONNECTED = -3,
+  INSTANCE_NORM = -2,
```

Comment:
Is it safe to change the enum values for circle-extended operator in term of backward compatibility?
I think the tflite_schema upversion is a rather big change. It may be good to use version (0.3->1.0) than 0.4 (0.3 -> 0.4).

Reply:
It is changed for backward compatibility. You can find `deprecated_builtin_code` is `byte` (not `ubyte`), so this change will confirm compatibility until TFLite don't introduce negative value enum operation.

Reply:
Thank you for pointing out `deprecated_builtin_code`. I can see the same code in master branch of tensorflow also.

https://github.com/tensorflow/tensorflow/blob/89486218bd8c476c7f5ba037de0f11b76d99c4be/tensorflow/lite/schema/schema.fbs#L1114-L1118